frontend/src/store/modules/review/mutations.js

```diff
@@ -0,0 +1,60 @@
+import normalizer from '../../../services/common/normalizerService';
+import { STATUS_NONE } from '@/services/api/codes';
+
+export default {
+    ADD_USER: (state, user) => {
+        if (state.users.allIds.includes(user.data.id)) {
+            return;
+        }
+        const newUser = normalizer.normalize(user, {
+            first_name: '',
+            last_name: '',
+            avatar_url: ''
+        });
+        state.users.byId = Object.assign(
+            {},
+            state.users.byId,
+            newUser.byId
+        );
+        state.users.allIds.push(user.data.id);
+    },
+
+    ADD_REVIEW: (state, review) => {
+        const newReview = normalizer.normalize( { data: review }, {
```

Comment:
This can be separate function `getReviewSchema()` inside review/state.js. This is duplicated few times
{
            created_at: '',
            description: '',
            dislikes: 0,
            like: STATUS_NONE,
            likes: 0,
            user_id: null
}